src/flavors/pbdurham/config.json

```diff
@@ -626,6 +596,7 @@
     "location_item_name": "location",
     "place_detail": [
       {
+        "surveyId": "culture",
```

Comment:
Note: The `surveyId` in our "place detail" configs isn't being used yet, but I think we should eventually migrate over to referencing the survey in each place detail config. This will decouple the survey form logic from the place detail presentation logic.

I added the attributes here as a reminder that they should be referencing the survey, instead of including the survey logic here. But I didn't make that refactor because I didn't want to pull that thread yet....